editor.planx.uk/src/lib/applications.ts

```diff
@@ -0,0 +1,49 @@
+import gql from "graphql-tag";
+import { client } from "lib/graphql";
+
+export type ApplicationData = {
+  session_id: string;
+  submitted_at: Date | string;
+  user_invited_to_pay: boolean;
+  payment_status: string;
+  amount: number;
+  payment_date: Date | string;
+  sent_to_email: boolean;
+  sent_to_bops: boolean;
+  sent_to_uniform: boolean;
+};
+
+export async function fetchSubmittedApplications(
+  flowSlug: string,
+  teamSlug: string,
+) {
+  const { data } = await client.query({
+    query: gql`
+      query SubmittedApplications($service_slug: String!, $team_slug: String!) {
+        applications_summary(
+          where: {
+            service_slug: { _eq: $service_slug }
+            team_slug: { _eq: $team_slug }
+            submitted_at: { _is_null: false }
```

Comment:
nit: I think this third `where` clause is unnecessary as you're already doing this in the view definition https://github.com/theopensystemslab/planx-new/pull/2867/files#diff-887c1fcbaafca5d3d84a6324752b4d8edb1fba21b9f9525ede41d6782f394f36R27

Reply:
Great spot! I had this before I decided to add the filter in the view. I'll get this updated 👍